backend/compact-connect/tests/unit/test_app.py

```diff
@@ -24,6 +24,10 @@ def test_no_compact_jurisdiction_name_clash(self):
             context = json.load(f)['context']
         jurisdictions = set(context['jurisdictions'])
         compacts = set(context['compacts'])
+        for jurisdiction in jurisdictions:
+            self.assertNotIn('#', jurisdiction, "'#' not allowed in jurisdiction abbreviations!")
+        for compact in compacts:
+            self.assertNotIn('#', compact, "'#' not allowed in compact abbreviations!")
```

Comment:
The `#` character is used in the composite identifiers in the database. In order to prevent confusion in parsing the identifiers, we would either have to carefully escape all `#` characters that might show up in them or simply not allow them in compacts. Since the compact abbreviations seem unlikely to include a `#` character, the latter seemed reasonable.

Reply:
Got it! Can you add a brief comment in this test explaining that?